Controller/Download/Attachment.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types = 1);
+declare(strict_types=1);
```

Comment:
Sprawdź wszystkie pliki *.php i popraw wszędzie `declare` tak jak tutaj

Reply:
All refactor-like notes will be corrected in separate PR, as whole module needs to be refactored (it is quite old). I will skip it in scope of this PR.